vaadin-multi-select-mixin.html

```diff
@@ -0,0 +1,149 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+<link rel="import" href="vaadin-list-mixin.html">
+<script>
+  window.Vaadin = window.Vaadin || {};
+
+  /**
+   * A mixin for `nav` elements, facilitating multiple selection of childNodes.
+   *
+   * @polymerMixin
+   */
+  Vaadin.MultiSelectMixin = superClass => class VaadinMultiSelectMixin extends Vaadin.ListMixin(superClass) {
```

Comment:
`MultiSelectMixin` is still a list mixin, right (because it extends it)? Shouldn't it say `MultiSelectListMixin`?

Reply:
Have renamed to `MultiSelectListMixin`